src/helpers/actions.ts

```diff
@@ -85,25 +91,24 @@ export const clickOnLogo = async (page: Page): Promise<void> => {
  * @returns true if found and updated, false otherwise
  */
 export const typeOnInputField = async (
-  page: Page,
+  page: DappeteerPage,
   label: string,
   text: string,
   clear = false,
   excludeSpan = false,
   optional = false
 ): Promise<boolean> => {
-  let input: ElementHandle<HTMLInputElement>;
+  let input: DappeteerElementHandle;
   try {
     input = await getInputByLabel(page, label, excludeSpan, 1000);
   } catch (e) {
     if (optional) return false;
     throw e;
   }
 
-  if (clear)
-    await page.evaluate((node: HTMLInputElement) => {
-      node.value = "";
-    }, input);
+  if (clear) {
+    await input.type("");
+  }
   await input.type(text);
```

Comment:
not sure if type `await input.type("");` will clear the input field

Reply:
modified respective implementations to delete on empty value